config/settings_example.py

```diff
@@ -58,7 +58,13 @@
                             'num_devices': 4,
                         }
                     ],
-                    'features': [FEATURES['CPU'], FEATURES['GPU']],
+                    'features': [
+                        FEATURES['CPU'],
+                        FEATURES['GPU'],
+                    ],
+                    'extras': {
+                        'gpu_vendor': GPU_VENDORS['NVIDIA'],
```

Comment:
introduce `GPU_VENDOR` + `NVIDIA` constants?

```suggestion
                        GPU_VENDOR: GPU_VENDORS[NVIDIA],